client/db.json

```diff
@@ -0,0 +1,16 @@
+{
+  "projects": [
+    {
+      "data": {
+        "image": "https://res.cloudinary.com/kschmitz/image/upload/v1606483223/upload/iyvfricfqui4dw2knkcv.jpg",
+
+        "projectTitle": "test",
+        "description": "beschreibung",
+        "tags": "tag, tag, ttag",
+        "category": "macrame"
+      },
+      "tags": ["tag", "tag", "ttag"],
```

Comment:
Why are these doubled?
Is one for the image and one for the project itself?

Reply:
i haven't found a way to format the tags with react-hook-form so i added a second, formatted line. But i'm happy about any suggestions on how to solve this :) (didn't work with useSate